tequilapi/client/client.go

```diff
@@ -129,6 +130,25 @@ func (client *Client) RegisterIdentity(address, beneficiary string, stake, fee u
 	return nil
 }
 
+// TopUp tops up identity
+func (client *Client) TopUp(identity string) error {
+	payload := transactor.TopUpRequest{
+		Identity: identity,
+	}
+
+	response, err := client.http.Post("transactor/topup", payload)
```

Comment:
since the accountant id is node-wide configurable string which is already a prerequisite to run node after the new payments are launched, it is already pre-configured and we don't need to ask for the accountant specifically.

Reply:
Approving for now but will have to be more flexible in the future.